octobot_backtesting/collectors/exchanges/exchange_collector.pxd

```diff
@@ -26,8 +26,19 @@ cdef class ExchangeDataCollector(data_collector.DataCollector):
     cdef public object tentacles_setup_config
     cdef public object start_timestamp
     cdef public object end_timestamp
+    cdef public bint finished
+    cdef public bint in_progress
+    cdef public int current_time_frame_index
+    cdef public int total_time_frame
+    cdef public int current_time_frame_percent
```

Comment:
This percentage cannot be floating?

Reply:
int this implementation it's only int. i'll change it now if someone create it's own